FirebaseMessaging/Sources/FIRMessagingConstants.m

```diff
@@ -18,13 +18,31 @@
 
 NSString *const kFIRMessagingFromKey = @"from";
 
-NSString *const kFIRMessagingMessageIDKey = @"gcm."
-                                            @"message_id";
+NSString *const kFIRMessagingSendTo = @"google."
+                                      @"to";
```

Comment:
I'm curious why it's split into two lines?

Reply:
This is like this long time ago, I can refactor in another PR.